### ISSUE TITLE
update SoundEffects to use tinysound instead of javax.sound

### ISSUE THREAD

**Issue by melvinzhang** [magarena/magarena#600]
possibly also consolidate or merge with MagicSound as both perform similar function

**Comment by lodici** [comment]
MagicSound is for UI sound effects and SoundEffects for gameplay. Merging seems very sensible.

**Comment by lodici** [comment]
Also SoundEffects use the "au" format which I had never heard of before where-as MagicSound uses wav. Consolidating to a common format would also be useful.

**Comment by lodici** [comment]
_TLDR: TinySound not used anymore; MagicSound replaces SoundEffects class; all sound files consolidated in jar; \release\sounds deleted; volume settings simplified and can be adjusted for both UI and game sounds._

On Linux Mint 17.3, I found that TinySound would clash with PulseAudio and effectively freeze the desktop. It would also play sounds distorted - the attached zip is an mp3 of the bong sound played on the missing image alert at startup. Of course this could be a configuration issue with my setup but it is the default setup I got after installing Mint. Moreover, the gameplay sounds play absolutely fine.

[wrong-bong.zip](https://github.com/magarena/magarena/files/76658/wrong-bong.zip)

As a result I replaced calls to TinySound with the same code used in the ex-SoundEffects class which is used for gameplay sounds. Initially, everything seemed fine except for some reason the bong sound was being truncated. It would appear I had stumbled across [this bug](http://bugs.java.com/bugdatabase/view_bug.do?bug_id=4434125). The solution was to increase the length of the bong wav by [adding some silence](https://github.com/magarena/magarena/commit/577bbd2a3384cfbd28b863c2a7222adeee8513ec).

Finally, I still found that playing sounds rapidly (eg. clicking on card in deck editor) did not work, with sounds going missing. The solution I found was to replace ``clip.stop()`` with ``clip.loop(0)`` which was inspired by [this article](http://raspberrypi.stackexchange.com/questions/14406/audio-in-java-application-wont-stop-when-told). Now sounds play immediately and in quick fire succession.

Tested on Windows 7 and Linux Mint 17.3.
